src/sentry/integrations/github/integration.py

```diff
@@ -98,19 +98,24 @@ def get_repositories(self, query=None):
     def search_issues(self, query):
         return self.get_client().search_issues(query)
 
-    def get_stacktrace_link(self, repo, filepath, version):
-        try:
-            self.get_client().check_file(repo.name, filepath, version)
-        except ApiError as e:
-            if e.code != 404:
-                raise
-            return None
-
+    def format_source_url(self, repo_name, filepath, branch):
         # Must format the url ourselves since `check_file` is a head request
         # "https://github.com/octokit/octokit.rb/blob/master/README.md"
-        web_url = u"https://github.com/{}/blob/{}/{}".format(repo.name, version, filepath)
+        return u"https://github.com/{}/blob/{}/{}".format(repo_name, branch, filepath)
+
+    def get_stacktrace_link(self, repo, filepath, default, version):
```

Comment:
@MeredithAnya shouldn't we put this method on the `RepositoryMixin` class so we don't have to duplicate this over each integration?